signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/SignInWithAppleService.kt

```diff
@@ -3,15 +3,35 @@ package com.willowtreeapps.signinwithapplebutton
 import android.net.Uri
 import android.os.Parcel
 import android.os.Parcelable
+import android.view.View
+import androidx.annotation.IdRes
 import androidx.fragment.app.FragmentManager
+import com.willowtreeapps.signinwithapplebutton.view.SignInWebViewDialogFragment
 import java.util.*
 
 class SignInWithAppleService(
-    private val clientId: String,
-    private val redirectUri: String,
-    private val scope: String
+    @IdRes private val id: Int,
+    private val fragmentManager: FragmentManager,
+    private val config: SignInWithAppleConfig,
+    private val callback: (SignInWithAppleResult) -> Unit
 ) {
 
+    constructor(
+        @IdRes id: Int,
+        fragmentManager: FragmentManager,
+        config: SignInWithAppleConfig,
+        callback: SignInWithAppleCallback
+    ) : this(id, fragmentManager, config, callback.toFunction())
+
+    init {
+        val fragmentIfCreated =
+            fragmentManager.findFragmentByTag(fragmentTag) as? SignInWebViewDialogFragment
```

Comment:
This will be non-null if there was a call to `show` on the prior instance, but we're not expressing that relationship clearly. How about we rename this to `fragmentIfShown`?